winfetch.ps1

```diff
@@ -678,20 +678,34 @@ foreach ($item in $config) {
 
         $output += "$($line["content"])"
 
-        # move cursor to column 40
         if ($img) {
-            $output = "$e[40G$output"
-            $writtenLines++
+            if (-not $stripansi) {
+                # move cursor to column 40
+                $output = "$e[40G$output"
+            } else {
+                # write image progressively
+                $imgline = "$($img[$writtenLines])".PadRight(35) -replace $ansiRegex, ""
```

Comment:
I think you don't need to replace ansiRegex here. It is anyways getting replaced in L695 below.

Reply:
haha yeah missed that